src/UI/SignTransactionsModals/SignWithLedgerModal/SignStep.tsx

```diff
@@ -131,6 +134,14 @@ const SignStep = ({
         <React.Fragment>
           {currentTransaction.transaction && (
             <React.Fragment>
+              {allTransactions.length > 1 && (
```

Comment:
This could be pulled into a named variable for clarity, like areMultipleTransactions or such